spec/models/rook_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+require 'rook'
+
+RSpec.describe Rook, type: :model do
+	before do 
+		@game = Game.create
+		@game.clear_board_pieces
+		@rook = Rook.create(:game => @game, :color => 0, :x_position => 0, :y_position => 0)
+
+	end
+
+	it "should allow a horizontal move" do
+		expect(@rook.valid_move?(0, 6)).to eq(true)
+	end
+
+	it "should allow a vertical move" do
+    	expect(@rook.valid_move?(6, 0)).to eq(true)
+	end
+
+	it "should only allow a move that is horizontal or vertical" do
```

Comment:
try to be a bit more descriptive and use 'should' and 'should not' in the naming, i.e. 

it "should not allow ***"